Flaky test_system_clusters_actual_information

https://s3.amazonaws.com/clickhouse-test-reports/0/332b3e644b8df1356da413b5429f154c7b564893/integration_tests__tsan__[4_6].html


```
>       error = node.query_and_get_error(
            "SELECT count() FROM distributed SETTINGS receive_timeout=1, handshake_timeout_ms=1"
        )

test_system_clusters_actual_information/test.py:43: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
helpers/cluster.py:3423: in query_and_get_error
    return self.client.query_and_get_error(
helpers/client.py:36: in wrap
    return func(self, *args, **kwargs)
helpers/client.py:135: in query_and_get_error
    ).get_error()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <helpers.client.CommandRequest object at 0x7ff387776f20>

    def get_error(self):
        self.process.wait(timeout=DEFAULT_QUERY_TIMEOUT)
        self.stdout_file.seek(0)
        self.stderr_file.seek(0)
    
        stdout = self.stdout_file.read().decode("utf-8", errors="replace")
        stderr = self.stderr_file.read().decode("utf-8", errors="replace")
    
        if (
            self.timer is not None
            and not self.process_finished_before_timeout
            and not self.ignore_error
        ):
            raise QueryTimeoutExceedException("Client timed out!")
    
        if self.process.returncode == 0:
>           raise QueryRuntimeException(
                "Client expected to be failed but succeeded! stdout: {}".format(stdout),
                self.process.returncode,
                stderr,
            )
E           helpers.client.QueryRuntimeException: Client expected to be failed but succeeded! stdout: 0
```

--- a/tests/integration/test_system_clusters_actual_information/test.py
+++ b/tests/integration/test_system_clusters_actual_information/test.py
@@ -12,20 +12,11 @@ cluster = ClickHouseCluster(__file__)
 node = cluster.add_instance(
     "node", with_zookeeper=True, main_configs=["configs/remote_servers.xml"]
 )
-node_1 = cluster.add_instance("node_1", with_zookeeper=True)
-
 
 @pytest.fixture(scope="module")
 def started_cluster():
     try:
         cluster.start()
-        node_1.query_with_retry("DROP TABLE IF EXISTS replicated")
-
-        node_1.query_with_retry(
-            """CREATE TABLE replicated (id UInt32, date Date) ENGINE =
-            ReplicatedMergeTree('/clickhouse/tables/replicated', 'node_1')  ORDER BY id PARTITION BY toYYYYMM(date)"""
-        )
-
         node.query_with_retry(
             "CREATE TABLE distributed (id UInt32, date Date) ENGINE = Distributed('test_cluster', 'default', 'replicated')"
         )
@@ -37,8 +28,6 @@ def started_cluster():
 
 
 def test(started_cluster):
-    cluster.pause_container("node_1")
-
     node.query("SYSTEM RELOAD CONFIG")
     error = node.query_and_get_error(
         "SELECT count() FROM distributed SETTINGS receive_timeout=1, handshake_timeout_ms=1"
@@ -67,5 +56,3 @@ def test(started_cluster):
 
     assert recovery_time == 0
     assert errors_count == 0
-
-    cluster.unpause_container("node_1")

--- a/tests/integration/test_system_clusters_actual_information/test.py
+++ b/tests/integration/test_system_clusters_actual_information/test.py
@@ -13,6 +13,7 @@ node = cluster.add_instance(
     "node", with_zookeeper=True, main_configs=["configs/remote_servers.xml"]
 )
 
+
 @pytest.fixture(scope="module")
 def started_cluster():
     try:

